csvcubed/csvcubed/utils/log.py

```diff
@@ -0,0 +1,39 @@
+"""
+Log
+---
+
+Utilities to help with logging.
+"""
+import logging
+import sys
+from colorama import Fore, Style
```

Comment:
I don't think this meets a threshold for inclusion as a dependency in our package. I suggest removing it.

Reply:
I mean if we can do the same without it then fine we can remove it, but I'm inclined to include it until we come across a problem requiring us to remove it.